pkgs/development/python-modules/tubes/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildPythonPackage, fetchPypi
+, characteristic, six, twisted
+}:
+
+buildPythonPackage rec {
+  pname = "Tubes";
+  version = "0.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256:0sg1gg2002h1xsgxigznr1zk1skwmhss72dzk6iysb9k9kdgymcd";
+  };
+
+  propagatedBuildInputs = [ characteristic six twisted ];
+
+  checkInputs = [ twisted ];
+
```

Comment:
```suggestion
```

Reply:
I thought this should be the case but when I leave twisted out of checkInputs the build fails:

```
running install tests
/nix/store/b39yxip6di5gq3q8513hfvmhfwz4yq77-stdenv-linux/setup: line 1315: trial: command not found
```

Reply:
I guess it has something to do with python modules vs bin contents.  I can side-step it by using `python -m` though, I guess...